feature-configs/ci/nmstate/operator_subscription.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  labels:
+    operators.coreos.com/kubernetes-nmstate-operator.openshift-nmstate: ""
+  name: kubernetes-nmstate-operator
+  namespace: openshift-nmstate
+spec:
+  channel: stable
+  installPlanApproval: Automatic
+  name: kubernetes-nmstate-operator
+  source: art-nightly-operator-catalog
+  sourceNamespace: openshift-marketplace
+---
```

Comment:
the `---`  goes to the top
```
yamllint  feature-configs/ci/nmstate/kustomization.yaml
feature-configs/ci/nmstate/kustomization.yaml
  1:1       warning  missing document start "---"  (document-start)
```

Reply:
just noticed all the yaml under `feature-configs` does not have `---` . Lets make it as all the other and live with that until we decide to put yamllint in the CI. Does it sound good?